cunumeric/deferred.py

```diff
@@ -1032,25 +1087,25 @@ def convert(self, rhs, warn=True):
         lhs = lhs_array.base
         rhs = rhs_array.base
 
-        task = self.context.create_task(CuNumericOpCode.CONVERT)
+        task = self.context.create_auto_task(CuNumericOpCode.CONVERT)
```

Comment:
Please review these blanket change. None of the calls pass `manual=True` so they should all definitely be `AutoTask`. Additionally, some of the resulting tasks get passed parameters that can only exist on `AutoTask` (e.g. `partition`)

Reply:
looks reasonable